lib/compiler/aro/aro/Preprocessor.zig

```diff
@@ -430,12 +431,24 @@ fn preprocessExtra(pp: *Preprocessor, source: Source) MacroError!TokenWithExpans
                         if_level = sum;
 
                         if (try pp.expr(&tokenizer)) {
-                            if_kind.set(if_level, until_endif);
+                            std.mem.writePackedInt(
+                                u2,
+                                &if_kind,
+                                if_level * 2,
+                                until_endif,
+                                native_endian,
+                            );
```

Comment:
It's a small thing, but you could use `writePackedIntNative` here to save a line in each instance.